CHANGELOG.md

```diff
@@ -38,8 +42,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 (sorry, we didn't keep a proper changelog previous to this release)
 
-[Unreleased]: https://github.com/appsembler/xblock-launchcontainer/compare/v2.3.1...HEAD
-[2.3.1]: https://github.com/appsembler/xblock-launchcontainer/compare/v2.3.1...HEAD
+[Unreleased]: https://github.com/appsembler/xblock-launchcontainer/compare/v2.3.2...HEAD
+[2.3.2]: https://github.com/appsembler/xblock-launchcontainer/compare/v2.3.2...HEAD
+[2.3.1]: https://github.com/appsembler/xblock-launchcontainer/compare/v2.3.1...v2.3.2
```

Comment:
For some reason all of the changelog diff links are off in this project. Ie, the link for release 2.3.1 should show a diff between 2.3.1 and 2.3.0, (the changes that went into 2.3.1) not a diff between 2.3.1 and 2.3.2.

Go ahead and merge this release though and we'll fix up the whole changelog in one separate commit.